src/DisplayPanel/DisplayControls.ts

```diff
@@ -106,24 +106,27 @@ function setOpacityControls (meiClassName: string): void {
   function inputChangeOpacity (): void {
     opacityOutput.value = opacitySlider.value;
     lastGlyphOpacity = Number(opacitySlider.value);
-    (document.querySelector('.' + meiClassName) as HTMLElement)
-      .style.opacity = (Number(opacityOutput.value) / 100.0).toString();
+    try {
+      ((document.querySelectorAll('.' + meiClassName)) as NodeListOf<HTMLElement>).forEach(g => {
+        g.style.opacity = (Number(opacityOutput.value) / 100.0).toString();
+      });
+    } catch (e) {}
```

Comment:
If this shouldn't fail silently, add a `console.warn` so debugging isn't painful later if.

Reply:
s/if/on
Always have coffee before doing a code review.